static/src/javascripts/projects/common/modules/commercial/build-page-targeting.spec.ts

```diff
@@ -2,22 +2,16 @@ import { cmp as cmp_ } from '@guardian/consent-management-platform';
 import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
```

Comment:
This spec arguably belongs in `commercial-core` now..

Reply:
Agree, we should do at some point in the future. Since we've got code coverage monitoring on commercial-core it'd be cool to see how  well-tested this function is.

Reply:
😬 

![Screenshot 2022-10-06 at 12 41 20](https://user-images.githubusercontent.com/7423751/194303791-20c9455b-89ea-4e87-a182-0424b709f38b.png)